http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/PersistentDiskComponent.scala

```diff
@@ -89,6 +155,23 @@ object persistentDiskQuery extends TableQuery(new PersistentDiskTable(_)) {
       }
     } yield (disk, label)
 
+  def updateGalaxyDiskRestore(id: DiskId, galaxyDiskRestore: GalaxyDiskRestore): DBIO[Int] =
+    findByIdQuery(id)
+      .map(x => (x.galaxyPvcId, x.cvmfsPvcId, x.chart, x.release))
+      .update(
+        (Some(galaxyDiskRestore.galaxyPvcId),
+         Some(galaxyDiskRestore.cvmfsPvcId),
+         Some(galaxyDiskRestore.chart),
+         Some(galaxyDiskRestore.release))
+      )
+
+  def isUsedByGalaxy(id: DiskId)(implicit ec: ExecutionContext): DBIO[Option[Boolean]] =
```

Comment:
Wondering if this should return `DBIO[Boolean]`.. is `Option` important?

Reply:
this function will actually be removed in the follow up PR